APR-Core/QuestHandler.lua

```diff
@@ -1773,3 +1779,58 @@ APR_QH_EventFrame:SetScript("OnEvent", function(self, event, ...)
         APR.party:SendGroupMessage()
     end
 end)
+
+function CheckIfPartyMemberIsFriend()

```

Comment:
move CheckIfPartyMemberIsFriend in Party.lua file 
+ rename it `APR.party:CheckIfPartyMemberIsFriend()`